infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -141,6 +130,12 @@ public LogUnitServer(ServerContext serverContext) {
         logCleaner = new StreamLogCompaction(streamLog, 10, 45, TimeUnit.MINUTES, ServerContext.SHUTDOWN_TIMER);
     }
 
+
+    @Override
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651438)